### ISSUE TITLE
Patch file document with data archive result

### ISSUE THREAD

**Issue by prabh-t** [ebi-ait/dcp-ingest-central#669]
After ingest data archiver has completed transferring data files for a submission to ENA upload area, the result of the transfer is published to the `ingest.data.archiver.complete` queue.

This result needs to the processed and the file metadata document needs to be updated in ingest with the outcome of the transfer, including the MD5 computed, if the file was compressed during the process, etc. This is needed before submitting the RUN XML ([ebi-ait/dcp-ingest-central#530](https://app.zenhub.com/workspace/o/ebi-ait/dcp-ingest-central/issues/530) ).

Component where change is needed:
- `ingest-data-archiver`
- `ingest-core` add new property to File entity

Patch to `file` document:
```
"fileArchiveResult": {
    "lastArchived":  datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
    "compressed": file.compressed,
    "md5": file.md5,
    "enaUploadPath": file.ena_upload_path,
    "error": file.error
}
```

**Comment by prabh-t** [comment]
PRs:
https://github.com/ebi-ait/ingest-core/pull/115 (approved, merged)
https://github.com/ebi-ait/ingest-data-archiver/pull/7 (approved, merged)

**Comment by Wkt8** [comment]
Still waiting on a review for second PR - then can be merged into prod.

**Comment by ESapenaVentura** [comment]
Still waiting on PR review!

**Comment by prabh-t** [comment]
Comments addressed. Waiting for (re-)re-approval..
